roles/nvidia-network-operator/tasks/k8smaster.yaml

```diff
@@ -0,0 +1,25 @@
+---
+# git clone; helm install; create node policy; create networkattachdefinition
+#
+
+- name: Git clone
+  ansible.builtin.git:
+    repo: 'https://github.com/Mellanox/network-operator.git'
+    dest: /tmp/network-operator
+
+- name: Deploy network-operator chart from local path
+  community.kubernetes.helm:
+    name: network-operator
+    chart_ref: /tmp/network-operator/deployment/network-operator/
```

Comment:
Why are we storing things in /tmp? We should try to download things to the local deepops dir (~/deepops) or into the {{ deepops_dir }} location (/opt/deepops).

In fact, I think a better way to do this overall would be to follow the pattern I set with the nfs-client-provisioner. In that case I added the external repo as a git submodule (https://github.com/NVIDIA/deepops/blob/master/.gitmodules), this gets downloaded into the submodules directory (at a specific commit) during setup. After that I run all the helm install commands using localhost, rather than downloading the files to the master nodes and then expecting them to have the right version of helm install and also having RW in /tmp.

Reply:
See https://github.com/NVIDIA/deepops/blob/master/roles/nfs-client-provisioner/tasks/main.yml for an example on how I delegated localhost to run helm install for the cloned repos.

If the git clone is not necessary, it would also be best-practice to do a helm repo add/helm repo update in place of a git clone (see the GPU Device Plugin role for an example of this).